### PR TITLE
fix(DB/item_template_locale)(Part Two) Fixing special characters for translation esES/esMX

### PR DESCRIPTION
**Pull request by arielcami**
## Changes Proposed:
-  Continued translation and special characters fix for esES and esMX locales

## Issues Addressed:

- Fix special character errors when using locales

## How to Test the Changes:

1.  .additem 47010
2.  Watch it's name and description in esES or esMX
3.  Run this SQL to fix this
ERRORES

## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.

### CODE REVIEW

File: data/sql/updates/pending_db_world/rev_1621427242858236100.sql [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+UPDATE `item_template_locale` SET `name` = 'Boceto: rubí cárdeno sutil' WHERE (`ID` = 46922) AND `locale` IN ('esMX','esES');
+UPDATE `item_template_locale` SET `name` = 'Boceto: ópalo crepuscular enjundioso' WHERE (`ID` = 41702) AND `locale` IN ('esMX','esES');
+UPDATE `item_template_locale` SET `name` = 'Boceto: ópalo crepuscular real' WHERE (`ID` = 41701) AND `locale` IN ('esMX','esES');
+UPDATE `item_template_locale` SET `name` = 'Boceto: ópalo crepuscular regio' WHERE (`ID` = 41703) AND `locale` IN ('esMX','esES');
+UPDATE `item_template_locale` SET `name` = 'Boceto: ojo de dragón rápido' WHERE (`ID` = 42307) AND `locale` IN ('esMX','esES');
+UPDATE `item_template_locale` SET `name` = 'Boceto: lágrima de pesadilla' WHERE (`ID` = 49112) AND `locale` IN ('esMX','esES');
+UPDATE `item_template_locale` SET `Description` = 'Te enseña a tallar un ametrino de precisión.' WHERE (`ID` = 47010) AND `locale` IN ('esMX','esES');
+UPDATE `item_template_locale` SET `Description` = 'Te enseña a tallar un ametrino destellante.' WHERE (`ID` = 47010) AND `locale` IN ('esMX','esES');
```
Comment (Helias) [inline data/sql/updates/pending_db_world/rev_1621427242858236100.sql:27]: you are overriding the texts, you should choose one of the two

Comment (Helias) [reply data/sql/updates/pending_db_world/rev_1621427242858236100.sql:27]: moreover you could merge the updates description and name in one line if you want, to have less code (less code usually means less possible errors)

Comment (arielcami) [reply data/sql/updates/pending_db_world/rev_1621427242858236100.sql:27]: Thank you so much for noticing, ill fix it right away!

**Comment by Helias** [comment]
you don't need to close the PR to update it, just update your branch and push the commits, they will be available also here @arielcami

**Comment by arielcami** [comment]
> you don't need to close the PR to update it, just update your branch and push the commits, they will be available also here @arielcami

I'm not sure how to do it, I'll talk with someone who knows.

**Review by Helias — APPROVED** [review]

**Comment by Helias** [comment]
@arielcami, I think it's ok now

**Pull request merged**